lib/commands/internal.js

```diff
@@ -1,16 +1,35 @@
+const logger = require('../utils/logger');
+
 class InternalCommand {
-	interactive() {}
-	make() {}
-	help() {
-		const HelpGroup = require('../groups/help');
-		const HelpOutput = new HelpGroup().run();
-		console.log(HelpOutput.outputOptions.help);
-	}
-	version() {
-		const jsonPath = require('path').join(__dirname, '..', '..', 'package.json')
-		const version = require(jsonPath, 'utf8').version
-		console.log(`webpack-cli ${version}`);
-	}
+    /**
+     *
+     * @param {String} command
+     * @param {any} args
+     */
+    constructor(command, ...args) {
+        /**
+         *
+         * @type {String}
+         */
+        this.command = command;
+        /**
+         *
+         * @type {any[]}
```

Comment:
@evenstensberg here we need to understand what's inside `args`.
Also, we should change the name because it's misleading. Maybe `cliArguments`?

Reply:
thought the same thing, you can rename it, invoked from `bootstrap.js` and it's missleading indeed